pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR.pkgproj

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="dir.props" />
+
+  <PropertyGroup>
+    <Version>2.0.0</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <File Include="files\Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR.props">
```

Comment:
The VS targets will handle this, but we'll need them to be ported to this package as well. The reason I have not done that yet is because we also need to port one or two MSBuild tasks and I thought that work could wait for the very-short term.